src/domain/auth/authenticate.ts

```diff
@@ -50,6 +48,19 @@ export const authenticateWithBackend = (
 
     dispatch(fetchTokenSuccess(res.data));
   } catch (error) {
-    dispatch(fetchTokenError(error));
+    toast(i18n.t('authentication.errorMessage'), {
```

Comment:
You forgot to add this error message to `fi.json` file.

Reply:
Fixed :)